lms/static/sass/course/ccx_coach/_dashboard.scss

```diff
@@ -44,12 +49,99 @@ table.ccx-schedule {
     border: 1px solid #cbcbcb;
     padding: 15px;
     margin-bottom: 20px;
+    button {
+        display: inline-block;
+        vertical-align: middle;
+        margin-right: .3125rem;
```

Comment:
`@include` syntax should always be used. See https://github.com/edx/edx-platform/wiki/RTL-UI-Best-Practices

Be sure to test your feature fully for internationalization and RTL. See http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#coverage-testing for instructions on how to do both parts.